pysrc/pip_resolve.py

```diff
@@ -20,7 +20,7 @@
         raise ImportError(
             "Could not import pkg_resources; please install setuptools or pip.")
 
-PYTHON_MARKER_REGEX = re.compile(r'python_version\s*(?P<operator>==|<=|=>|>|<)\s*\'(?P<python_version>.+)\'')
+PYTHON_MARKER_REGEX = re.compile(r'python_version\s*(?P<operator>==|<=|=>|>|<)\s*[\'"](?P<python_version>.+?)[\'"]')
```

Comment:
Supporting double quotes also for python markers and making the python_version group match non-greedy to avoid blowing up by attempting to convert `6\' or python_version >= \'2` into a number.